src/protagonist/API/Features/NamedQueries/Converters/NamedQueryConverter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using API.Exceptions;
+
+namespace API.Features.NamedQueries.Converters;
+
+public static class NamedQueryConverter
+{
+    public static DLCS.HydraModel.NamedQuery ToHydra(this DLCS.Model.Assets.NamedQueries.NamedQuery namedQuery, string baseUrl)
+    {
+        var hydraNamedQuery = new DLCS.HydraModel.NamedQuery(baseUrl, namedQuery.Customer, namedQuery.Id)
+        {
+            Name = namedQuery.Name,
+            Template = namedQuery.Template,
+            Global = namedQuery.Global
+        };
+        return hydraNamedQuery;
+    }
+    
+    public static DLCS.Model.Assets.NamedQueries.NamedQuery ToDlcsModel(this DLCS.HydraModel.NamedQuery hydraNamedQuery, int customerId)
+    {
+        hydraNamedQuery.CustomerId = customerId;
```

Comment:
This is an unexpected/undocumented side-effect of calling this method. As a caller all I'm expecting is to be given a DLCS NamedQuery back, not for my HydraModel to be changed.

It'd be better to set customerId on the `HydraModel` prior to calling `ToDlcsModel()`, or update the object initialisation of DLCS model to set `Customer = customerId` directly.

Reply:
This is a very good point - I'm not sure why I did this, but I may have been trying to follow the format seen in `AssetConverter.cs`. Will fix.

Reply:
This has now been addressed in 7e88d073fbe143a69ffaca7e61071ab75d98cb71, c02b5bea60d44bd0ef8144290f1220f2f5864efe and 7e88d073fbe143a69ffaca7e61071ab75d98cb71